### ISSUE TITLE
log.setLevel() method ignored

### ISSUE THREAD

**Issue by c0lin** [darrylwest/simple-node-logger#4]
The `setLevel()` method seems to fail [while setting property `level` directly in `opts` does not]:

**Fails** with only Warning and Info logged
```
var log = require('simple-node-logger').createSimpleFileLogger('testLogger.log');
log.setLevel('all');
log.warn("Warning");
log.info("Info");
log.debug("Debug");
log.trace("Trace");
```

**Passes** with Warning, Info, Debug, and Trace logged
```
var opts = {logFilePath: 'testLogger.log', level: 'all'};
var log = require('simple-node-logger').createSimpleFileLogger(opts);
log.warn("Warning");
log.info("Info");
log.debug("Debug");
log.trace("Trace");
```

Tested on Node v0.10.25 and v4.2.1.

**Comment by slattery** [comment]
I believe I've addressed this in PR #7

**Comment by darrylwest** [comment]
thanks for the fix
